website-custom-views/src/content/development/forms.mdx

```diff
@@ -429,181 +429,6 @@ const ChannelsDetails = (props) => {
 }
 ```
 
-## Using modal pages
```

Comment:
Was this removed on purpose?

Reply:
Hi @emmenko, I removed it because the [`@application-components`](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-components#formmodalpage) page is available in the Custom Application docs but not in Custom Views docs. So I thought this doesn't apply here. If this is not the case, should we bring the (/api-reference/commercetools-frontend-application-components) page and this section back?

Reply:
I think this is because we don't want to have nested pages in Custom Views panels.

Reply:
Thanks @CarlosCortizasCT for clarifying this.

As a follow up question: does it mean that Custom Views shouldn't use different routes?

Reply:
I'm not sure we have though is such a use case until last week when someone raised this potential requirement.

Currently we support routing and navigation could be implemented but we don't expect modals to be used.

@mmaltsev-ct do you think this is something we must discuss about for the MVP?

Reply:
@CarlosCortizasCT, no more changes required here right?

Reply:
No, thanks 👍